README.md

```diff
@@ -86,6 +86,16 @@ sudo make install
 
 `runc` will be installed to `/usr/local/sbin/runc` on your system.
 
+#### Version string customization
+
+You can see the runc version by running `runc --version`. You can append a custom string to the
+version using the VERSION_EXTRA variable for make when you compile, e.g.:
+```bash
+make VERSION_EXTRA="build-1"
+```
+The runc version printed will be appended with `$(VERSION_EXTRA)`. Bear in mind to include some
+separator for readability.
+
```

Comment:
I think that "The runc version printed will be appended with `$(VERSION_EXTRA)`." is kind of redundant here; I'd remove it.

Also, your example doesn't follow your own advice about adding a separator.

Reply:
haha, true, I didn't update the example to include it now that we removed it from the Makefile. Updated that and removed the redundant sentence, thanks!